backend/app/api/endpoints/adapter/tasks.py

```
@@ -851,8 +859,23 @@ async def export_task_docx(
     - Embedded images and attachment info
     """
+    from app.services.auth.task_token import verify_task_token
     from app.services.task_member_service import task_member_service
 
+    if download_token:
+        token_info = verify_task_token(download_token)
+        if token_info is None or token_info.task_id != task_id:
+            raise HTTPException(status_code=401, detail="Invalid download token")
+        current_user = (
+            db.query(User)
+            .filter(User.id == token_info.user_id, User.is_active == True)
```

Ruff (0.16.3)

[error] Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)